vcr/issuer/issuer.go

```diff
@@ -215,8 +215,9 @@ func (i issuer) buildVC(ctx context.Context, template vc.VerifiableCredential, o
 		ExpirationDate:    template.ExpirationDate,
 		IssuanceDate:      template.IssuanceDate,
 	}
-	if unsignedCredential.IssuanceDate.IsZero() {
-		unsignedCredential.IssuanceDate = TimeFunc()
+	if unsignedCredential.IssuanceDate == nil || unsignedCredential.IssuanceDate.IsZero() {
```

Comment:
the `IsZero` was needed to check that no value was provided. If this true now, the zero value was provided by the user and it should not be changed but return an error? (from the validator later on)

Reply:
True. Would be a functionally invalid value, but user-provided anyway (or due to another bug, but then this "fix" obscures that bug), will revert.